### DISCUSSION TITLE
When can we expect the new version ?

### DISCUSSION THREAD

**Discussion by PiotrPrus** [getsentry/sentry-kotlin-multiplatform#162] [General]
Hi, we are using Sentry in our iOS and Android projects and we tried the sentryKMP as well. We love it and would like to use it daily, but the current version `0.3.0` uses quite old version of cocoa sentry. We are waiting for new version of sentry KMP to use it in the project :) 
Thanks for your work to make KMP version ❤️

**Comment by buenaflor** [comment]
Hi!

We will release another version very very soon that uses the latest cocoa (8.17.1) sdk :)

**Reply by buenaflor** [reply]
Just a small update, we will release `0.4.0` this week (without performance monitoring)

**Reply by buenaflor** [reply]
`0.4.0` is now released: https://github.com/getsentry/sentry-kotlin-multiplatform/releases

**Comment by alexwhb** [comment]
@buenaflor is there ETA on a new version? I saw that you guys did a merge request adding in the features I requested. 🎉

**Reply by buenaflor** [reply]
`0.5.0` is out now!

**Reply by alexwhb** [reply]
Absolutely no worries. Thank you so much. Hope you get well soon. Really excited to have these new features.
